eradiate/radprops/rad_profile.py

```diff
@@ -376,8 +376,7 @@ def to_dataset(self, spectral_ctx=None):
                 z_level=self.levels,
                 sigma_t=self.sigma_t().flatten(),
                 albedo=self.albedo().flatten(),
-            )
-
+            ).isel(w=0)
```

Comment:
Yes indeed the spectral coordinate becomes a scalar.
I'm not 100% sure but I don't think it's a problem.

In these data sets, when the mode is monochromatic, the wavelength coordinate is pretty useless at the moment. It is there only so that the data set format allows to store spectra.